instrumentation/spring-amqp/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>io.zipkin.brave</groupId>
+		<artifactId>brave-instrumentation-parent</artifactId>
+		<version>4.14.4-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>brave-instrumentation-spring-amqp</artifactId>
+	<name>Brave Instrumentation: Spring AMQP Listener and Template</name>
+
+	<properties>
+		<main.basedir>${project.basedir}/../..</main.basedir>
+		<main.java.version>1.6</main.java.version>
+		<main.signature.artifact>java16</main.signature.artifact>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.amqp</groupId>
+			<artifactId>spring-amqp</artifactId>
+			<version>${spring-amqp.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
```

Comment:
add a comment why aop is needed (usually we don't depend on other libraries that aren't direct dependencies)